backends/qualcomm/tests/test_qnn_delegate.py

```diff
@@ -561,7 +543,7 @@ def __init__(self):
                 self.avgPool = torch.nn.AvgPool2d(
                     kernel_size=(2, 2),
                     padding=(1, 1),
-                    stride=(2, 2),
+                    stride=(1, 1),
```

Comment:
why this change

Reply:
We notice this test parameter is break in some version of QNN. Change to a simpler case to make unit test focus on the framework, not the SDK itself.